scripts/bulk-delete.bash

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+echo Enter your JupiterOne accountId
+read J1_ACCOUNT_ID
+
+echo Enter your JupiterOne API token
+read J1_API_TOKEN
+
+echo Enter the J1QL query to get the list of entities to be deleted. For example: Find aws_instance with _source='system-mapper' 
+read J1_QUERY
+
+echo Executing...
+
+j1 \
+  -q "$J1_QUERY as e return e._id as entityId" \
+  -a $J1_ACCOUNT_ID -k $J1_API_TOKEN \
+  --output-file /tmp/j1-temp-entities.json
```

Comment:
It could be worthwhile, with such a destructive action, to say something like "Please ensure the results in /tmp/j1-temp-entities.json are safe to delete! There's no going back. Are you sure? Y/[N]:".

Reply:
+1

Reply:
yep, i can add in a future PR